Config/ConfigLoader.php

```diff
@@ -47,9 +49,17 @@ public function loadFromAdditionalConfiguration(): void
     {
         $this->contentElement->load();
 
-        // remove TYPO3 error and exception handler to use Symfony instead
-        $GLOBALS['TYPO3_CONF_VARS']['SYS']['productionExceptionHandler'] = '';
-        $GLOBALS['TYPO3_CONF_VARS']['SYS']['debugExceptionHandler'] = '';
+        if ($_SERVER['APP_DEBUG']) {
+            // remove TYPO3 error and exception handler to use Symfony instead, if in DEBUG mode
+            $GLOBALS['TYPO3_CONF_VARS']['SYS']['debugExceptionHandler'] = '';
+            $GLOBALS['TYPO3_CONF_VARS']['SYS']['productionExceptionHandler'] = '';
+        } else {
+            // use custom TYPO3 exception handler if not in DEBUG mode to fix the Output Buffer issue of
+            // the TwigBundle and BartacusTwigBundle
+            $GLOBALS['TYPO3_CONF_VARS']['SYS']['debugExceptionHandler'] = Typo3DebugExceptionHandler::class;
+            $GLOBALS['TYPO3_CONF_VARS']['SYS']['productionExceptionHandler'] = Typo3ProductionExceptionHandler::class;
+        }
+
```

Comment:
Can you please extract this block into it's own config loader class? I never made interfaces nor kernel tags for that, but that would be quite nice